test/formatDate.spec.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatDate from "../dist/date";
+
+Globalize.load(
+    require( "cldr-data/main/en/ca-gregorian" ),
+    require( "cldr-data/main/en/timeZoneNames" ),
+    require( "cldr-data/main/en/numbers" ),
+    require( "cldr-data/supplemental/likelySubtags" ),
+    require( "cldr-data/supplemental/timeData" ),
+    require( "cldr-data/supplemental/weekData" )
+);
+
+describe("formatDate Component", () => {
+    it("renders as a <span>", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date()}</FormatDate>);
+        expect(wrapper.type()).to.eql("span");
+    });
+
+    it("formats date as 1/1/2016", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date("Jan 01 2016")}</FormatDate>);

```

Comment:
It is rebuilt on every run so `dist` is up to date. I have changed locally to use `src` and all tests pass though I have to remove lines like this https://github.com/jquery-support/react-globalize/blob/master/src/currency.js#L3 which I am honestly not sure why they would be there and what exactly they would be importing. After removal though, all tests pass.